app/commands/exercise/representation/trigger_reruns_for_track.rb

```diff
@@ -0,0 +1,12 @@
+class Exercise::Representation::TriggerRerunsForTrack
```

Comment:
I've extracted this logic from `HandleDeploy` to this command. Seemed a bit cleaner.